docs/competition/safeguarding_2020/safeguarding_2020.md

```diff
@@ -0,0 +1,41 @@
+# Safeguarding 2020
+
+Unforeseeable circumstances in SR2020 has created a situation where we need to give further thought to safeguarding as our normal processes no longer  cover all eventualities.
+
+This guidance is purely for SR2020, and must be reviewed for applicability and appropriateness before any future events.
+
+This guidance is largely driven by the fact that we can no longer guarantee the involvement of the competitor's responsible adult in any interactions.
+
+## YouTube comments
+
+Most of the communication during the SR2020 competition is likely to be through comments on the livestream YouTube video.
+
+All normal advice regarding suitability of comments should be observed, in addition the following should be followed:
+
+- Personal contact details/social media accounts should not be shared or used.
+- No private discussions should be held between volunteers and competitors.
+- In the unlikely event that a competitor has to be contacted privately, another adult must be included in the discussion, ideally this should be both the competitor's responsible adult and another SR volunteer.
+- In the event that a competitor makes contact privately, these additional parties must be immediately included.
+
+## Competitor Interviews
+
+Some pre-recorded interviews with competitors are going to be displayed during the events.
```

Comment:
So far, no team has volunteered to do these interviews so it's a possibility rather than a definite

Reply:
Moot. The policy needs to be in place before they can occur.
If they don't occur, they don't occur. But we can't do this stage after they've happened, so it needs to be ready in expectation of.